test/functional/verify/latest-missing/test.bats

```diff
@@ -5,7 +5,7 @@ load "../../swupdlib"
 @test "verify install using latest with missing version file on server" {
   run sudo sh -c "$SWUPD verify $SWUPD_OPTS --install -m latest"
 
-  check_lines "$output"
```

Comment:
Why aren't we checking the output any more? Are we losing the error message?

Reply:
Good question.

This test is checking the behavior of swupd when the latest file was missing in the server. Before this patch CA Paths weren't used when downloading the latest file, so the error messages have changed. The problem is that the output will be different if the code is compiled with or without CA paths and looked enough. I haven't realized though that this is easy to fix using reg exp. I'll fix that and check the output correctly.